modules/cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -49,19 +49,24 @@ spec:
       rfc: RFC5424
       severity: debug
      url: 'udp://rsyslogserver.west.example.com:514'
+  inputs: <8>
```

Comment:
This input is not used anywhere under pipelines as an 'inputRef'. I suggest to use it instead of 'application' tenant.

Reply:
hi @kabirbhartiRH I'm not sure I understand what you mean. could you please explain your point further?

Reply:
ohh ok thank you for elaborating. I will make the change

Reply:
@pfeifferj I still don't see the respective change. Could you check once.